### PR TITLE
Acordian

### PR DESCRIPTION
**Pull request by ZackBradshaw**

### CODE REVIEW

File: studio/package.json [hunk 1]
```diff
@@ -24,8 +24,8 @@
     "eslint": "^8.6.0",
     "prop-types": "^15.7",
     "react": "^17.0",
+    "react-burger-menu": "^3.0.8",
```
Comment (nathansbradshaw) [inline studio/package.json]: Why did you add this to the studio?

File: web/components/Footer.module.scss [hunk 2]
```diff
@@ -3,55 +3,50 @@
 .container {
     display: flex;
     position: absolute;
-    width: 100%;
     height: 18vh;
     background-color: $background;
     align-items: center;
-    justify-content: center;
+    // justify-content: center;
```
Comment (nathansbradshaw) [inline web/components/Footer.module.scss]: is there a reason that you are keeping this comment?

File: web/components/Layout.module.scss [hunk 3]
```diff
@@ -1,13 +1,18 @@
...
+    width: 100%;
+}
 @media only screen and (max-width: 1024px) {
     .container{
        width: 100vw; 
     }
-}
\ No newline at end of file
+
+}
```
Comment (nathansbradshaw) [inline web/components/Layout.module.scss]: It's good practice to leave a blank line at the end of the file.
```suggestion
}

```

File: web/components/Layout.tsx [hunk 4]
```diff
@@ -20,9 +20,11 @@ const Layout = ({ children, title = "ZackBradshaw" }: Props) => {
...
-        {children}
-        <Footer />
+        <div className={styles.flex}>
+          <Navbar />
+          {children}
+          <Footer />
+        </div>
       </div>
     </div>
   );
```
Comment (nathansbradshaw) [inline web/components/Layout.tsx]:
```suggestion
  );
  
```

File: web/components/Navbar.tsx [hunk 5]
```diff
@@ -29,7 +41,46 @@ const Navbar = () => {
...
+            <div className={styles.link}>
+              <span className={styles.itemIcon}>#</span>
+              <p>contacts</p>
+            </div>
+          </Link>
+        </li>
+      </ul>
     </div>
   );
 };
```
Comment (nathansbradshaw) [inline web/components/Navbar.tsx]:
```suggestion
};

```

File: web/components/SectionHeader.module.scss [hunk 6]
```diff
@@ -3,11 +3,17 @@
...
         color: $pink;
         font-size: 30px;
     }
+}
+
+.line{
+    background-color: $pink;
+    height: .2rem;
+    width: 30;
 }
```
Comment (nathansbradshaw) [inline web/components/SectionHeader.module.scss]:
```suggestion
}

```

File: web/package.json [hunk 7]
```diff
@@ -10,9 +10,11 @@
     "@portabletext/react": "^2.0.0",
     "@sanity/image-url": "^1.0.1",
     "groq": "^2.33.2",
+    "hamburger-react": "^2.5.0",
     "next": "latest",
     "next-transpile-modules": "^9.1.0",
     "react": "^18.2.0",
+    "react-burger-menu": "^3.0.8",
```
Comment (nathansbradshaw) [inline web/package.json]: Whu did you pull in 2 hamburger libraries? are you using them both?

File: web/styles/reset.scss [hunk 8]
```diff
@@ -57,4 +62,12 @@ table {
...
 scroll-behavior: smooth!important;
+}
+
+:before, *:after {
+    box-sizing: border-box;
+}
+
+#__next {
+    width: 90vw;
 }
```
Comment (nathansbradshaw) [inline web/styles/reset.scss]:
```suggestion
}

```

**Review by nathansbradshaw — CHANGES_REQUESTED** [review]
Add a gitignore file to the studio folder, and fix the few changes I have suggested, other than that it looks good. (Note I did not run the code, and in most cases I won't so make sure everything works before a review)

File: web/package.json (hunk 7 above)
Comment (ZackBradshaw) [reply web/package.json]: im using none of them i was going to use them but then i opted for creating my own from scratch

File: web/components/Footer.module.scss (hunk 2 above)
Comment (ZackBradshaw) [reply web/components/Footer.module.scss]: not any good ones removing that now

File: studio/package.json (hunk 1 above)
Comment (ZackBradshaw) [reply studio/package.json]: was in the wrong folder when i npm installed

File: web/styles/reset.scss [hunk 9]
```diff
@@ -57,4 +62,12 @@ table {
...
\ No newline at end of file
+}
+
+:before, *:after {
+    box-sizing: border-box;
+}
+
+#__next {
+    width: 90vw;
+}
```
Comment (nathansbradshaw) [inline web/styles/reset.scss]:
```suggestion
}

```

File: web/styles/projects.module.scss [hunk 10]
```diff
@@ -24,11 +24,12 @@
...
+    gap: 1rem;
 }
 
 .projects{
     display: flex;
-    gap: 30px;
+    gap: 1rem;
     flex-wrap: wrap;
+    justify-content: center;
 }
```
Comment (nathansbradshaw) [inline web/styles/projects.module.scss]:
```suggestion
}

```

File: web/components/SectionHeader.module.scss [hunk 11]
```diff
@@ -3,11 +3,17 @@
...
     }
-}
\ No newline at end of file
+}
+
+.line{
+    background-color: $pink;
+    height: .2rem;
+    width: 30;
+}
```
Comment (nathansbradshaw) [inline web/components/SectionHeader.module.scss]:
```suggestion
}

```

**Review by nathansbradshaw — CHANGES_REQUESTED** [review]

**Review by nathansbradshaw — APPROVED** [review]
